src/sourceMaps/sourceMaps.ts

```diff
@@ -17,59 +17,34 @@ export class SourceMaps {
         this._sourceMapFactory = new SourceMapFactory(pathMapping, sourceMapPathOverrides, enableSourceMapCaching);
     }
 
-    /**
-     * Returns the generated script path for an authored source path
-     * @param pathToSource - The absolute path to the authored file
-     */
-    public getGeneratedPathFromAuthoredPath(authoredPath: string): string {
-        authoredPath = authoredPath.toLowerCase();
```

Comment:
Did this move or is it not needed anymore?

Reply:
We replaced this by the Script model and it's developmentSource, runtimeSource, mappedSource, and sourceMapper properties